besecure_developer_toolkit/cli.py

```diff
@@ -149,6 +152,24 @@ def report(
             raise typer.Exit()
 
 
+@validate.command("version_file")
+def version_data_naming_convention_validation(
+    issue_id: int = typer.Option(None, prompt="Enter OSSP id", help="OSSP id"),
+    name: str = typer.Option(None, prompt="Enter OSSP name", help="OSSP name"),
+    namespace: str = typer.Option(None, prompt="Enter GitHub username", help="namespace"),
+    branch: str = typer.Option(None, prompt="Enter branch", help="branch"),
+    ):
+    """ Check version details file naming convention """
+    version_data = VdncValidate(issue_id, name, namespace, branch)
```

Comment:
can you use some meaning full function name from current function I don't understand what it does

Reply:
updated